admin_guide/diagnostics_tool.adoc

```diff
@@ -298,6 +302,24 @@ A user-defined timeout may be set by passing the
 if a newly-created log entry is not able to be queried via Elasticsearch after
 45 seconds.
 
+|`sdn`
+a|This check performs the following cluster-level diagnostics of the
+{product-title} SDN:
+
+* Verify that master hosts can connect to kubelets.
+* Verify that nodes can route packets to each other.
+* Validate node addresses.
+* Validate HostSubnet objects.
+
+If you specify the `*openshift_checks_output_dir*` variable, the check will also
```

Comment:
s/will also save/also saves
Everything else LGTM. :)

Which version(s) is this change valid for @Miciah? (If you have a BZ or Trello with the info, you could link to it.)

@openshift/team-documentation, PTAL

Reply:
Fixed, thanks!

This is for OCP 3.11 onwards.

https://github.com/openshift/openshift-ansible/pull/7759
https://trello.com/c/HtaFZbiR
No Bugzilla report.

Reply:
Thank you for being so responsive! I'll hold onto this one until we open the 3.11 docs, then. :)